dependency.lic

```diff
@@ -202,6 +203,8 @@ class ScriptManager
     @shop_update_queue = []
 
     @enable_fork = LICH_VERSION =~ /f/
+
+    @versions = nil
```

Comment:
Now that I look at it, doing it this way prevents it from computing hashes for developer mode, but that's a debatable win.